Update README.md (#87)
OfficeDev/Office-Samples: Excel-HelloWorld-TaskPane-JS/README.md

### Run the add-in using Office Add-ins Development Kit extension
The simpliest way to run this add-in project is using the Office Add-ins Development Kit. The [Office Add-ins Development Kit](https://marketplace.visualstudio.com/items?itemName=msoffice.microsoft-office-add-in-debugger) is an end-to end developer tool for building Office add-ins. You can use this tool to easily creating, running and debugging an Office add-in.

1. **Install the Office Add-ins Development Kit**
    
    Install the [Office Add-ins Development Kit](https://marketplace.visualstudio.com/items?itemName=msoffice.microsoft-office-add-in-debugger) extension in Visual Studio Code marketplace. 

1. **Download the sample code**

   You can use either of the ways to download the sample code:

>    * In the `Office Add-ins Development Kit` extension, find the sample in `Sample gallery`, click the "create" button at the top of the sample page.

Reviewer: I think this describes the UI flow better. Also, UI options should be **bold** and we should favor "select" over "click" to support non-mouse inputs.
```suggestion
   * Open the Office Add-ins Development Kit extension. Select **View Samples** and choose a sample from the **Sample Gallery**. Select the **Create** button in the top-right corner of the sample page.
```

Author: I change the description as below:
Open the Office Add-ins Development Kit extension and view samples in the **Sample gallery**. Select the **Create** button in the top-right corner of the sample page.

The reason is that developers may see this page under different circumstances. For developers that already arrive at sample page in VSC from web link or sample gallery, they only need to create, so we might not emphasize on previous actions like Select View samples and choose a sample; For developers that starts from GitHub, they can still go to download and view samples in the sample gallery.

>    * Clone or download this sample to a folder on your computer, then open the folder in Visual Studio Code.

Reviewer: ```suggestion
   * [Clone](https://docs.github.com/repositories/creating-and-managing-repositories/cloning-a-repository) this repository or download this sample to a folder on your computer. Then, open the folder in Visual Studio Code.
```

   
1. **Open the Office Add-in Development Kit**
   

>    Click the <img src="./assets/Icon_Office_Add-ins_Development_Kit.png" width="30"/> button in the side panel to open the extension.

Reviewer: Use the same alt text in this image as the icon in the extension displays.
```suggestion
   Click the <img src="./assets/Icon_Office_Add-ins_Development_Kit.png" width="30" alt="Office Add-in Dev Kit"/> button in the side panel to open the extension.
```

Author: Add the alt text
